sys/dev/ath/if_ath_usb.c

```diff
@@ -1121,6 +1121,10 @@ ath_if_intr_tx_callback(struct usb_xfer *xfer, usb_error_t error)
 			wakeup(&usc->wait_msg_id);
 			STAILQ_INSERT_TAIL(&usc->sc_cmd_inactive, cmd, next);
 		}
+		if (usc->wait_msg_id == AR_WMI_CMD_MSG)
+		{
```

Comment:
Open bracket should be in one line with if statement.

Reply:
Done - brackets removed. Thx!